src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -2239,14 +2241,21 @@ void AuraEffect::HandleFeignDeath(AuraApplication const* aurApp, uint8 mode, boo
                 }
             }
         }
-        target->CombatStop();
+
+        if (target->GetMap()->IsDungeon()) // feign death does not remove combat in dungeons
```

Comment:
Does this mean that if the hunter is the last target alive during boss encounter and he uses feign death then boss doesn't evade with this change?